app/api/contests/contests.go

```diff
@@ -21,9 +21,12 @@ func (srv *Service) SetRoutes(router chi.Router) {
 	router.Use(render.SetContentType(render.ContentTypeJSON))
 	router.Use(auth.UserMiddleware(srv.Store.Sessions()))
 
-	router.Put("/contests/{item_id}/additional-time", service.AppHandler(srv.setAdditionalTime).ServeHTTP)
 	router.Get("/contests/{item_id}/group-by-name", service.AppHandler(srv.getGroupByName).ServeHTTP)
-	router.Get("/contests/{item_id}/members-additional-times", service.AppHandler(srv.getMembersAdditionalTimes).ServeHTTP)
+
```

Comment:
Should "/contests/{item_id}/group-by-name" be "/contests/{item_id}/groups/by-name" as well?

Reply:
fixed